sdk/servicebus/service-bus/src/track2/constructorHelpers.ts

```diff
@@ -11,6 +11,22 @@ import {
   ServiceBusClientOptions
 } from "../old/serviceBusClient";
 
+/**
+ * Parses a connection string and extracts the EntityPath named entity out.
+ * @param connectionString An entity specific Service Bus connection string.
+ * @internal
+ * @ignore
+ */
+export function getEntityNameFromConnectionString(connectionString: string): string {
```

Comment:
Love it. Probably a bad time to point out that core-amqp exports a function to parse connection strings: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-amqp/src/index.ts#L50

Reply:
Will fix in next PR! :)